neuralmonkey/encoders/recurrent.py

```diff
@@ -401,6 +429,7 @@ def rnn(self) -> Tuple[tf.Tensor, tf.Tensor]:
         each layer are used as inputs to the next one. As a final state of the
         stacked RNN, the final state of the final layer is used.
         """
+        # pylint: disable=redefined-variable-type
```

Comment:
Nemělo by se tady s tim pylintem spíš něco udělat, než ho disablenout?
Btw obecně když se disabluje nějakej pylint, u kterýho neni zřejmý proč, měl by tam bejt komentář, proč to nejde naprogramovat líp.

Reply:
A jakto že to jinde funguje?

Reply:
rnn_input na zacatku je instanemethod (to jsou ty vase dekoratory) a to, co vraci rnn_layer je tensor

Reply:
Takže je to pylintovej false positive. Tak to chce napsat do komentáře. Divný je, že si to před touhle změnou nestěžovalo. Taky prosím přidej pylint enable hned za tu chybu.